### DISCUSSION TITLE
Notification Implementations

### DISCUSSION THREAD

**Discussion by straussmaximilian** [straussmaximilian/tasklit#3] [Ideas]
A cool feature would be to integrate notifications of any sort, whenever a job is finished.
There are several packages that allow low-level entry to this

- [Teams](https://pypi.org/project/pymsteams/)
- [Slack](https://pypi.org/project/slackclient/)
- [Email](https://realpython.com/python-send-email/)
- [Telegram](https://pypi.org/project/telegram-send/)
- Signal?
- SMS? Might not be used by many?

**Comment by SecretLake** [comment]
Definitely. We can start with basic stuff, like teams, slack, etc. And then add chat messengers.
